### PR TITLE
get Actions working for check, coverage

### LINKED ISSUE #206: getting up and running with Actions
Travis is going away in a matter of weeks.

I think the immediate need is to get pkgdown working; this is discussed in #204 and addressed in #205 

Here, we want to get the rest of the CI up-and-running, using its own PR:

- [x] get R-CMD-Check running with Actions, without token
- [x] get R-CMD-Check running with Actions, with token
- [x] get test coverage running with Actions
- [x] disable other CI
- [x] update badges in README

### PR DESCRIPTION
**Pull request by ijlyttle**
fix #206

### CODE REVIEW

**Comment by ijlyttle** [comment]
I'm repeating myself...

**Comment by ijlyttle** [comment]
got API tests running Actions on Mac only, after much gnashing of teeth. 

It introduces another environment variable, `BOXR_IGNORE_TOKEN` which needs to be present and `false` for the token to be ignored - should run OK on local computer without it.

A walk has being suggested urgently, will resume.

**Comment by ijlyttle** [comment]
Hi Nate - this is ready for you to look at. Please let me know if it's OK, and I'll be happy to merge.

I'll make a note to write up the testing procedure as I work on the tests - there is a README.md there that could use updating.

In the meantime, you'll need to add `BOXR_USE_TOKEN="true"` to your `.Renviron` file to run the API tests.

File: .github/workflows/R-CMD-check.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: r-lib/actions/setup-r@v1
+        with:
+          r-version: ${{ matrix.config.r }}
+
+      - uses: r-lib/actions/setup-pandoc@v1
+
+      - name: Diagnostic
```
Comment (nathancday) [inline .github/workflows/R-CMD-check.yaml:46]: Do you want to keep this in there?

**Review by nathancday — APPROVED** [review]
You work fast!  Geez that is a lot of config for GH-actions, but it does seem to be doing it's job. One small nit about the Diagnostic step, but otherwise looks good to go.

File: .github/workflows/R-CMD-check.yaml (hunk 1 above)
Comment (ijlyttle) [reply .github/workflows/R-CMD-check.yaml]: nice catch on the Diagnostic, thanks! As you probably guessed, that was when I was twiddling with the extra env variable to make sure we run only once against the API.

**Comment by ijlyttle** [comment]
A lot of config, yes - so I am very happy Jim Hester has already sorted through all that :)

**Pull request merged**